openbrews/recipe/detail-edit/edit-recipe.html

```diff
@@ -50,6 +50,39 @@
                         <input type="text" name="mashEfficiency" ng-model="recipe.mashEfficiency" class="text-right">
                         <span class="input-addon">%</span>
                     </label>
+                    <label for="steepEfficiency" class="item item-input item-input-group">
+                        Steep Efficiency
+                        <input type="text" name="steepEfficiency"
+                        ng-model="recipe.steepEfficiency" class="text-right">
+                        <span class="input-addon">%</span>
+                    </label>
+
+                    <label for="og" class="item item-input">
+                        OG
```

Comment:
@mdw7326 Using the input tag here, lets the user independently change these values. Of course, if the user changes the dependencies then this value is updated. But at the end of the recipe, the user can simply set these values to something and then save the recipe, and it will be these values which will be saved.

Was this what you intended?